_posts/2023-02-01-solidity-0.8.18-release-announcement.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.18 Release Announcement'
+date: '2023-02-01'
+author: Solidity Team
+category: Releases
+---
+
+## Introducing the newest version of the Solidity Compiler!
+We are excited to announce the latest release of the Solidity Compiler, [Solidity v0.8.18](https://github.com/ethereum/solidity/releases/tag/v0.8.18). 
+This latest version includes a range of improvements and it also introduces support for the [Paris upgrade](https://blog.ethereum.org/2022/08/24/mainnet-merge-announcement)!
+
+
+### Noteworthy Features
+
+* A new command-line flag (`--no-cbor-metadata`) and Standard JSON option (`settings.metadata.appendCBOR: false`) to prevent compiler from appending the CBOR metadata section at the end of the bytecode. Until now, it was only possible to exclude the hash of the JSON metadata file (`--metadata-hash none`/`settings.metadata.bytecodehash: "none"`) from that section, but the bytecode would still include a small bit of CBOR data containing the version of the compiler.
+* The support to the Paris hard fork includes: 
+    * Deprecation of global `block.difficulty` and removal of `difficulty()` in inline assembly for EVM versions >= Paris. The change is due to the renaming and semantic changes of the `difficulty` instruction introduced by [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399).
+    * Introduction of global `block.prevrandao` in Solidity and `prevrandao()` in inline assembly for EVM versions >= Paris.
+* `selfdestruct` is now considered deprecated ([EIP-6049](https://eips.ethereum.org/EIPS/eip-6049)) and the compiler will warn about its use, both in Solidity and in Yul, including inline assembly. There is currently no replacement, but its use is highly discouraged because it will eventually change its semantics and all contracts using it will be affected in some way.
+* This release also involves a change in the generation of Yul code to EVM bytecode when enabling the optimizer: for Yul functions that never return back to the caller (since they always revert or always terminate the EVM call), we no longer push a return label to the stack like it is needed for other functions to jump back to the caller. This will decrease both gas cost and code size. Tooling that relies on analyzing our calling conventions on the bytecode level may need to be adjusted to accommodate this change.
```

Comment:
I'd tweak the wording a bit:
```suggestion
* This release also involves a change in the generation of EVM bytecode from Yul code when the optimizer is enabled: for Yul functions that never return back to the caller (i.e. always revert or always terminate the EVM call), the compiler no longer pushes a return label. The label is only used to jump back to the caller. This will decrease both gas cost and code size. Tooling that relies on the current calling convention on the bytecode level may need to be adjusted to accommodate this change.
```